ci/travis/build_projects.sh

```diff
@@ -4,7 +4,12 @@ set -e
 
 if [ "$TRAVIS_OS_NAME" == "linux" ] 
 then 
+	echo "Building AD9361-Linux"
```

Comment:
why this `echo "Building AD9361-Linux"` ? ; it sounds like the entire no-OS project is the `AD9361-Linux` project

Reply:
well, to maybe later, let's move this single project in a more organized way;
like maybe each project build should be it's own function;
something like `build_ad9361_linux()`; then we would have another `build_ad9371_generic()` function, etc

Reply:
now i am a bit confused;
why is this patch changing `projects` build code, when the commit description tells that it's handling `drivers` build?